manager/controllers/baseline_create.go

```diff
@@ -67,21 +70,20 @@ func CreateBaselineHandler(c *gin.Context) {
 		return
 	}
 
-	baselineID, err := buildCreateBaselineQuery(request, accountID)
+	resp, err := buildCreateBaselineQuery(request, accountID)
 	if err != nil {
 		LogAndRespError(c, err, "Database error")
 		return
 	}
 
 	configUpdated := request.Config != nil
-	inventoryIDs := kafka.GetInventoryIDsToEvaluate(&baselineID, accountID, configUpdated, nil)
-	kafka.EvaluateBaselineSystems(inventoryIDs)
+	inventoryIDs := kafka.GetInventoryIDsToEvaluate(&resp.BaselineID, accountID, configUpdated, nil)
+	go kafka.EvaluateBaselineSystems(inventoryIDs)
```

Comment:
@MichaelMraka what do you think about this one?

Reply:
One issue per commit. :)
If you have an idea for improvement please create an independent commit for it so it's easier do distinguish what is the primary fix/feature and what's the improvement.